Dockerfile

```diff
@@ -77,7 +77,7 @@ WORKDIR /
 
 RUN git clone https://github.com/coala/coala-bears.git
 WORKDIR /coala-bears
-RUN pip3 install -r requirements.txt -r test-requirements.txt
+RUN pip3 download -r requirements.txt -r test-requirements.txt
```

Comment:
This was added for #17 ; removing it would be a Revert, and then I loose the ability to easy&quickly test against both released bears and master bears using the same docker job.